### DISCUSSION TITLE
Shortcuts

### DISCUSSION THREAD

**Discussion by marek90** [isontheline/pro.webssh.net#620] [Ideas]
Hi, just an idea: Would it be possible to expose the interface of web SSH to the Shortcuts app? Whether mashREPL only, or even other saved connections to real servers. Something akin to Shortcuts "Run Script Over SSH" action. This would dramatically improve automation possibilities.

**Comment by isontheline** [comment]
Hi @marek90 👋

Thank you a lot for your idea!
There is an issue attempt here : #354 

Servers should be setup inside WebSSH before issuing shortcuts with password or private key in order run smoothly, do you agree?

**Reply by marek90** [reply]
Hi @isontheline 

Yes, the servers need to be setup and work inside WebSSH first. 
I see the issue #354, but it seems like a different problem / solution. 

What do you think about my original proposal? 
Or do you think it's the same as #354?

**Reply by isontheline** [reply]
You're right #354 doesn't covers all use cases.

I'm opening an issue with your proposal here : https://github.com/isontheline/pro.webssh.net/issues/650
